src/nxdoc/nuxeo-server/core-server/automation/automation-scripting.md

```diff
@@ -256,13 +256,12 @@ Nuxeo Automation Scripting allows you to write JavaScript code to use Automation
 *   Using operations/chains as JavaScript functions within this `run()` function
 
     ```js
-    // For instance, the operation to display JSF messages
-    WebUI.AddMessage(input, {
-        /*required:true - type: string*/
-        'message': "Display It!",
-        /*required:true - type: string*/
-        'severity': "INFO"
-    });
+    /*Automation operation to create a new version*/
+    input = Document.CreateVersion(input,{
```

Comment:
Can you format the code ?
- space before `{`
- 4 more spaces for children
- closing `});`